### DISCUSSION TITLE
global.inlineSpacing token question

### DISCUSSION THREAD

**Discussion by heath-capstone** [primefaces/community#1739] [Figma UI Kit] [answered]
Hello, 

I have a few questions about the` global.inlineSpacing` token that I was hoping someone might be able to help me with.  

1. what was the expected use for this token? 
2. where and how is it set?  
3. from a search it appears that it's only used in one place: `toast.detail.margin.top` 

Essentially I'm interested in having set "spacing tokens" and it appears that that's what `global.inlineSpacing` is potentially intended to do...but I'm not sure about that. See attached image for spacing tokens used in other design systems for an idea of what I'd like to do. 

Thanks! 

Heath

![image](https://github.com/primefaces/community/assets/156356168/3d614b76-fbee-466e-ad26-bfd501724a17)

**Comment by onursenture** [comment, marked as answer]
Hello,

Yes, `global.inlineSpacing` was a commonly used variable in many components in the past. I have confirmed that currently, it is only used for `toast.detail.margin.top`: 

https://github.com/primefaces/primevue-sass-theme/blob/360a66efb83c65887ada53dbd241cd89c4d93037/themes/lara/lara-light/_variables.scss#L587

I agree with your suggestion that spacing should be handled under the **Primitive** category with general spacing tokens. We are developing the new theming engine and future versions of PrimeOne using this principle.

Thank you for your feedback. 🌟
